src/enum_clike.rs

```diff
@@ -0,0 +1,56 @@
+//! lint on C-like enums that are repr(isize/usize) and have values that don't fit into an i32
+
+use rustc::lint::*;
+use syntax::ast::{IntTy, UintTy};
+use syntax::attr::*;
+use rustc_front::hir::*;
+use rustc::middle::const_eval::{ConstVal, EvalHint, eval_const_expr_partial};
+use rustc::middle::ty;
+use utils::span_lint;
+
+/// **What it does:** C-like enums that are `repr(isize/usize)`` and have values that don't fit into an `i32`
```

Comment:
There is one quote to many after `repr(isize/usize)` and other wiki entries end with a point :smile: